packages/tezos/tezos.9.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "ledgerwallet-tezos"
+  "tezos-node" { = version }
+  "tezos-client" { = version }
+  "tezos-signer" { = version }
+  "tezos-accuser-009-PsFLoren" { = version }
+  "tezos-baker-009-PsFLoren" { = version }
+  "tezos-endorser-009-PsFLoren" { = version }
+  "tezos-accuser-010-PtGRANAD" { = version }
+  "tezos-baker-010-PtGRANAD" { = version }
+  "tezos-endorser-010-PtGRANAD" { = version }
+  "tezos-codec" { = version }
+
+  "tezos-embedded-protocol-001-PtCJ7pwo" { = version }
+  "tezos-embedded-protocol-002-PsYLVpVv" { = version }
+  "tezos-embedded-protocol-003-PsddFKi3" { = version }
+  "tezos-embedded-protocol-004-Pt24m4xi" { = version }
+  "tezos-embedded-protocol-005-PsBABY5H" { = version }
+  "tezos-embedded-protocol-005-PsBabyM1" { = version }
+  "tezos-embedded-protocol-006-PsCARTHA" { = version }
+  "tezos-embedded-protocol-007-PsDELPH1" { = version }
+  "tezos-embedded-protocol-008-PtEdoTez" { = version }
+  "tezos-embedded-protocol-008-PtEdo2Zk" { = version }
+
+  "tezos-client-001-PtCJ7pwo-commands" { = version }
+  "tezos-client-002-PsYLVpVv-commands" { = version }
+  "tezos-client-003-PsddFKi3-commands" { = version }
+  "tezos-client-004-Pt24m4xi-commands" { = version }
+  "tezos-client-005-PsBabyM1-commands" { = version }
+  "tezos-client-006-PsCARTHA-commands" { = version }
+  "tezos-client-007-PsDELPH1-commands-registration" { = version }
+  "tezos-client-008-PtEdo2Zk-commands-registration" { = version }
+]
+synopsis: "Tezos meta package installing all active binaries"
+
+url {
+  src: "https://gitlab.com/tezos/tezos/-/archive/v9.1/tezos-v9.1.tar.bz2"
+  checksum: [
+    "sha256=114073c8038a3329158557cfec00d2c8a1fadac3b622b30f8ecf1479949028d0"
+    "sha512=79631f68ac61c139ff849f23b6a9301c090d6afec091bde855e05ec753151a0259d61e509b67162ff49a2c80872d5a1d1ec65ff32b5cf09b6ea4684e269e4fd7"
+  ]
+}
```

Comment:
```suggestion
}
```
This is the archive for tezos 9.1. Furthermore I don't think having an archive is required for this package

Reply:
I fully agree but I put them to turn of the warning "non conf packages with no sources" (or something similar) :-)

Reply:
Let's try to go with accurate opam metadata please -- we'll fix the CI. It exists to help us humans, not the other way around :-)